Fix:

error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'isalpha' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2002-03-08 13:55:12.000000000 -0600
+++ configure	2021-05-20 02:12:32.000000000 -0500
@@ -2687,6 +2687,7 @@
 #line 2687 "configure"
 #include "confdefs.h"
 #include <ctype.h>
+#include <stdlib.h>
 #if ((' ' & 0x0FF) == 0x020)
 # define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
 # define TOUPPER(c) (ISLOWER(c) ? 'A' + ((c) - 'a') : (c))
--- headers.h.orig	2002-03-08 13:18:20.000000000 -0600
+++ headers.h	2021-05-20 02:24:48.000000000 -0500
@@ -13,6 +13,7 @@
 #include <config.h>
 #include <custom.h>		/* to override anything in config.h */
 
+#include <ctype.h>
 #include <stdio.h>		/* NB: this must come BEFORE <sys/types.h>, */
 				/* because on DEC Alpha OSF/1 3.x with lcc, the */
 				/* reverse order fails because of a conflicting */
